modules/xfem/test/tests/corner_nodes_cut/tests

```diff
@@ -5,33 +5,47 @@
     exodiff = 'corner_node_cut_twice_out.e'
     map = false
     unique_id = true
+    requirement = 'The XFEM module shall have the capability of cutting '
+                  'through a quad element at the two nodes opposite to each '
+                  'other to split the element into two separate equally sized '
+                  'triangular elements.'
```

Comment:
The XFEM module shall have the capability of cutting directly through multiple nodes of a single QUAD4 element in 2D